scripts/modules-tests.sh

```diff
@@ -0,0 +1,38 @@
+# go into the directory we want to compile
+cd ./modules
+
+# Create lock file if it does not exist
+if [ ! -f Cargo.lock ]; then
+  cargo generate-lockfile
+fi
+
+# Force recompile osmosis test tube so build.rs is runned
+cargo clean -p osmosis-test-tube
```

Comment:
Maybe it's possible to instead cache `.so` files generated by `osmosis-test-tube`, but I'm done with CI for now